src/index.html

```diff
@@ -15,8 +15,56 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&amp;display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="/apple"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="/samsung">Samsung</a></li>
+          <li><a href="/smartphones">Smartphones</a></li>
+          <li>
+            <a
+              href="/laptops"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li><a href="/gadgets">Gadgets</a></li>
+          <li><a href="/tablets">Tablets</a></li>
+          <li><a href="/photo">Photo</a></li>
+          <li><a href="/video">Video</a></li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
For better accessibility, consider using semantic HTML5 elements such as `<nav>` for the navigation bar instead of a generic `<div>` or `<header>`. This will help screen readers and other assistive technologies understand the structure of the page better.